enhancements/windows-containers/windows-machine-config-operator.md

```diff
@@ -78,6 +77,7 @@ has a policy to not ship 3rd party kernels for support reasons.
 ### Design Details
 
 The WMCO image will be packaged with the following binaries:
+* Containerd
```

Comment:
@Anandnatraj containerd is lower case except this one place where it has been capitalized as the first letter of the bullet point.